test/NEW-IMAGES

```diff
@@ -12,3 +12,4 @@
 #
 # Format is one FQIN per line. Enumerate them below:
 #
+quay.io/libpod/testimage:20241011
```

Comment:
Shouldn't we get this into the CI image first? This will cause quay.io flakes...

Reply:
It already is in the latest CI images! (The zstd ones. Getting podman to work with zstd is just a SMOP).

Oh all right. I'll build new images with only that change...

Reply:
Yep I know it is annoying but I think we have established that we cannot pull from quay.io reliably enough for podman CI usage. It is not if but when it will flake, if I would know that the zstd image is ready quickly then I might be fine to carry this but given the amount of bugs I doubt it will happen fast enough.